src/CreateOverlay.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import OverlayMixin from './OverlayMixin';
+import RootCloseWrapper from './RootCloseWrapper';
+
+
+export default function createOverlayedComponent (render, { renderChildren = false, rootClose = true } = {}) {
```

Comment:
This isn't used anywhere and duplicates a lot of code from `Portal` &c. Do we need this?